product_pack/models/product_product.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api
+import math
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    pack_line_ids = fields.One2many(
+        'product.pack.line',
+        'parent_product_id',
+        'Pack Products',
+        help='Products that are part of this pack.'
+    )
+    used_pack_line_ids = fields.One2many(
```

Comment:
```suggestion
    used_in_pack_line_ids = fields.One2many(
```

I think this way the field usage is more self-explained. :thinking: